CHANGELOG-1.6.md

```diff
@@ -0,0 +1,64 @@
+# Changes since v1.5.0
+
+## Deprecations
+
+- Support for storage.k8s.io/v1beta1.CSINode and the nodeid annotation will be
+  removed in a future release. Only cluster versions that support
+  storage.k8s.io/v1.CSINode will be supported.
+  ([#385](https://github.com/kubernetes-csi/external-provisioner/issues/385),
+  [#257](https://github.com/kubernetes-csi/external-provisioner/issues/257))
+- The current leader election lock name will be changed in a future release.
+  Rolling upgrades to a newer sidecar will not maintain proper leadership, and a
+  full teardown and bringup will be required if the driver cannot handle
+  multiple leaders.
+  ([#295](https://github.com/kubernetes-csi/external-provisioner/issues/295))
+- The "--leader-election-type" argument will be removed in a future release.
+  Only Leases-type leader election will be supported. The
+  "--enable-leader-election" argument will also be renamed to
+  "--leader-election".
+  ([#401](https://github.com/kubernetes-csi/external-provisioner/issues/401))
+- The default value of "--worker-threads" will be lowered to 10 to match the
+  other CSI sidecars in a future release.
+  ([#322](https://github.com/kubernetes-csi/external-provisioner/issues/322))
+- The external-provisioner will no longer default an empty fstype to "ext4".
+  A new option to set the default will be added, otherwise the CSI driver
+  must be able to handle an empty fstype according to the CSI spec.
+  ([#328](https://github.com/kubernetes-csi/external-provisioner/issues/328))
```

Comment:
Why are there referencing issues, not PRs?

Reply:
The deprecations section is talking about behavior that we want to change in a future release, there are no PRs existing to implement them yet.